### DISCUSSION TITLE
What's the release schedule of Milvus2.3?

### DISCUSSION THREAD

**Discussion by lyajedi** [milvus-io/milvus#26237] [Q&A and General discussion] [answered]
I need some features from Milvus2.3 like range search, cuda support and complex data types, but I don't find any info when it's going to be released besides some outdated info in confluence, which is already over due to the plan.
Do you have a rough estimation when 2.3 will be released?
Is it ok to use 2.3Beta instead if no estimation, I'm using 2.2.8 now,

**Comment by yhmo** [comment]
Next week.

**Reply by xiaofan-luan** [reply, marked as answer]
Next tuesday

**Reply by lyajedi** [reply]
Hi, is the release schedule updated?

**Reply by yhmo** [reply]
> Hi, is the release schedule updated?

Yes, there are some important issues not fixed well, the release was denied. Postpone it again, to the next week.

**Reply by xiaofan-luan** [reply]
We will make the branch out today

**Reply by xiaofan-luan** [reply]
and another round of stability test will be issued

**Reply by wanjz08** [reply]
Hi, any update on when 2.3 will be released?

**Reply by yhmo** [reply]
The v2.3.0 docker image was released last night. We will update the doc later.
